### ISSUE TITLE
Eclipser 0.6.0 fails to covert launch configurations in Intellij 14

### ISSUE THREAD

**Issue by hetul** [kukido/eclipser#12]
Eclipser cannot convert launch configurations in Intellij 14, but can convert the same launch configuration in IntelliJ 13. Eclipser gives the following dialog box when attempting to convert a launch configuration:

> Info
> 
> Eclipser was unable to conver launch files(s).
> Please submit support ticket at
> https://github.com/kukido/eclipser/issues

Here is a sample launch configuration:

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication">
<listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
<listEntry value="/jetty6/src/com/flurry/jetty/JettyServer.java"/>
</listAttribute>
<listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
<listEntry value="1"/>
</listAttribute>
<booleanAttribute key="org.eclipse.debug.core.appendEnvironmentVariables" value="true"/>
<listAttribute key="org.eclipse.debug.ui.favoriteGroups">
<listEntry value="org.eclipse.debug.ui.launchGroup.debug"/>
<listEntry value="org.eclipse.debug.ui.launchGroup.run"/>
</listAttribute>
<stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="com.flurry.jetty.JettyServer"/>
<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="agentPortal"/>
</launchConfiguration>
```

**Comment by kukido** [comment]
Thanks! Will check.

**Comment by kukido** [comment]
Hi Hetul,
I tried to run your configuration against 0.6.0 and it converts without any issues. 

Are you converting one file or a group? Is the conversion failing from any IDE Panel (Project, Commander)? Does the team have the same issue?

Could you please get the latest version of the plugin - 0.6.2 and see if the problem persists.
Cheers,
Andrey

**Comment by hetul** [comment]
Hey Andrey,

With 0.6.0 it worked in Commander, but not when using the "Convert with Eclipser Hotkey" in the main window (with the file open). Trying the hotkey with the file open would result in the error message I posted above.

After updating to 0.6.2, it still works in Commander. I also checked the project window and it works there too. However, the hotkey still doesn't work in the main window. Instead of showing an error, nothing appears. It seemed to work the first time I tried it, but did not work on subsequent tries, so maybe my configuration is somehow messed up? Note that the hotkey works in both the project and commander windows, but not when executed on the open file itself.

**Comment by kukido** [comment]
Hi Hetul,
I was able to replicate the issue. I've never tried to run conversion with hotkey, so was not aware of the issue, thank you for bringing it up. In 0.6 Eclipser was changed to support multiple files and with processing action from a hotkey, the event's structure is different. I'm going to add the code to handle those actions properly. Please stay tuned.

**Comment by kukido** [comment]
Hey Hetul,
Bug is fixed in 0.6.3 - please update and let me know if the issue still persists.
Cheers,
Andrey
